cmd/skaffold/app/flags/image.go

```diff
@@ -37,22 +37,40 @@ type image struct {
 }
 
 // String Implements String() method for pflag interface and
-// returns a comma separated list of images.
+// returns a placeholder for the help text.
 func (i *Images) String() string {
+	return strings.Join(i.GetSlice(), ",")
+}
+
+// Type Implements Type() method for pflag interface
+func (i *Images) Type() string {
+	return fmt.Sprintf("%T", i)
+}
+
+// SetNil Implements SetNil() method for our Nillable interface
+func (i *Images) SetNil() error {
+	i.images = []image{}
+	return nil
+}
+
+// Set Implements Set() method for pflag interface
+func (p *Images) Set(csv string) error {
+	split := strings.Split(csv, ",")
+	return p.Replace(split)
```

Comment:
I realized that this should be `.Append()` to maintain compatibility with our previous behaviour where we required used to provide multiple `-i` flags, one for for each image